server/app/views/components/ProgramQuestionBank.java

```diff
@@ -188,6 +189,27 @@ private FormTag questionBankPanel() {
                 h2("All other questions").withClasses(AdminStyles.SEMIBOLD_HEADER))
             .with(each(nonUniversalQuestions, qd -> renderQuestionDefinition(qd))));
 
+    contentDiv.with(
+        div()
+            .with(
+                Icons.questionTypeSvgWithId(QuestionType.ADDRESS),
+                Icons.questionTypeSvgWithId(QuestionType.CURRENCY),
+                Icons.questionTypeSvgWithId(QuestionType.CHECKBOX),
+                Icons.questionTypeSvgWithId(QuestionType.DATE),
+                Icons.questionTypeSvgWithId(QuestionType.DROPDOWN),
+                Icons.questionTypeSvgWithId(QuestionType.EMAIL),
+                Icons.questionTypeSvgWithId(QuestionType.ENUMERATOR),
+                Icons.questionTypeSvgWithId(QuestionType.FILEUPLOAD),
+                Icons.questionTypeSvgWithId(QuestionType.ID),
+                Icons.questionTypeSvgWithId(QuestionType.NAME),
+                Icons.questionTypeSvgWithId(QuestionType.NUMBER),
+                Icons.questionTypeSvgWithId(QuestionType.RADIO_BUTTON),
+                Icons.questionTypeSvgWithId(QuestionType.STATIC),
+                Icons.questionTypeSvgWithId(QuestionType.TEXT),
+                Icons.questionTypeSvgWithId(QuestionType.PHONE),
+                Icons.questionTypeSvgWithId(QuestionType.NULL_QUESTION))
```

Comment:
Can you create a program and manually verify all the different types still load correctly?

Reply:
Done! works pretty well :) ever faster